### DISCUSSION TITLE
Welcome to dooboo-ui Discussions!

### DISCUSSION THREAD

**Discussion by hyochan** [hyochan/dooboo-ui#8] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hyochan** [comment]
`dooboo-ui` aims to

1. Maintain reusable UI components
2. Unify code structures
3. Create a UI framework that works seamlessly on all platforms (web, iOS, Android)
4. Share the creativity
5. Highly customizable with minimal effort
6. Organize theme in ease

**Comment by yujonglee** [comment]
I aims to do these things,

1. Refactor existing codebase to make it easy to understand and maintain.
2.  Write expressive, self-explainary test codes.
3.  Find powerful abstracts to enhance the overall functionality and reusability.

**Reply by hyochan** [reply]
Looking forward to it! 🌞
